Source/WebCore/platform/graphics/nicosia/texmap/NicosiaGCGLANGLELayer.h

```diff
@@ -41,7 +41,9 @@ namespace WebCore {
 class IntSize;
 class GraphicsContextGLANGLE;
 class GraphicsContextGLFallback;
+#if USE(LIBGBM)
```

Comment:
Not strictly necessary, I've left them out in my WPE test builds -- but it's fine as-is.